src/Raven.Server/Config/Categories/SubscriptionConfiguration.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+using Raven.Server.Config.Attributes;
+using Raven.Server.Config.Settings;
+
+namespace Raven.Server.Config.Categories
+{
+    public class SubscriptionConfiguration : ConfigurationCategory
+    {
+        [DefaultValue(10)]
```

Comment:
Apparently it's not that easy.. I've used the help of 5 of our teammates, and managed to run 1000 concurrent subscriptions, with a pretty good throughput (don't have the exact number, but it's more than a hundred empty documents per subscription. I think that I'll set the value to 1000 for now. Something else that bothers me is the client side cost: every 100 subscription connections consumes ~25% of the cpu and ~4 gigs of memory, but I think that we'll leave that to RavenDB-7283 (subscriptions benchmark issue)